### DISCUSSION TITLE
Do bUnit tests needed?

### DISCUSSION THREAD

**Discussion by Andras-Csanyi** [microsoft/fluentui-blazor#249] [Ideas]
Hey All,

I was thinking adding bUnit tests to this project. In my opinion, they are useful (can check whether the rendered html fits for expectation, can check some functionality too, etc.). On the other hand, can dust off my rusty Blazor skills.

If you guys see value in it, I'm happy to provide them. What do you think?

**Comment by vnbaaij** [comment]
Would definitely be a good enhancement! Have not gotten round of adding these kind of tests myself so PR's would be appreciated.

**Reply by Andras-Csanyi** [reply]
Here is the first PR: https://github.com/microsoft/fast-blazor/pull/250
